addons/website_quote/views/sale_quote_views.xml

```diff
@@ -7,7 +7,12 @@
             <field name="arch" type="xml">
                 <form string="Sale Quotation Template">
                     <sheet>
-                        <button string="Edit Template" type="object" name="open_template" class="oe_highlight oe_right"/>
+                        <div class="oe_button_box" name="button_box">
+                            <button name="toggle_active" type="object" class="oe_stat_button" icon="fa-archive">
+                                <field name="active" widget="boolean_button" options='{"terminology": "archive"}'/>
+                            </button>
+                            <button string="Edit Template" type="object" name="open_template" class="oe_stat_button" icon="fa-pencil"/>
+                        </div>
```

Comment:
Can you add a search view with an "Archived" filter allowing to find archived quotation templates? Thanks